src/current/cockroachcloud/resource-usage-basic.md

```diff
@@ -64,24 +66,27 @@ If any queries are more expensive than expected, you can use the [`EXPLAIN ANALY
 
 Each query has an associated RU cost, so the total number of queries is an important factor in your consumption.
 
-To diagnose excessive queries, navigate to your cluster's **Metrics** page, [**Monitor SQL Activity** tab]({% link cockroachcloud/metrics-monitor-sql-activity.md %}) in the {{ site.data.products.cloud }} Console. The **SQL Statements** chart displays the number of queries over time. Look for any spikes or increases in QPS (queries per second) that may correspond to increases in your resource consumption.
+To diagnose excessive queries, navigate to your cluster's [**Overview** metrics page]({% link cockroachcloud/metrics-overview.md %}) in the {{ site.data.products.cloud }} Console. The [**SQL Statements** chart]({% link cockroachcloud/metrics-overview.md %}#sql-statements) displays the number of queries over time. Look for any spikes or increases in QPS (queries per second) that may correspond to increases in the [**Request Units** chart]({% link cockroachcloud/metrics-overview.md %}#request-units).
```

Comment:
I am keeping it as `**Overview** metrics page` because it is the type of metrics on the page: "Overview metrics", similar to "SQL metrics" and "Changefeed metrics".